cmd/inspector/main.go

```diff
@@ -244,6 +253,47 @@ func NodeInfo(cmd *cobra.Command, args []string) (err error) {
 	return nil
 }
 
+func dotOutput() (*os.File, error) {
+	if DotPath == "stdout" {
+		return os.Stdout, nil
+	}
+	if DotPath == "" {
+		DotPath = "routing-graph.dot"
+	}
+
+	return os.Create(DotPath)
+}
+
+// DrawTableAsGraph outputs the table routing as a graph
+func DrawTableAsGraph(cmd *cobra.Command, args []string) (err error) {
+	i, err := NewInspector(*Addr, *IdentityPath)
+	if err != nil {
+		return ErrInspectorDial.Wrap(err)
+	}
+
+	info, err := i.kadclient.DrawTable(context.Background(), &pb.DrawTableRequest{})
```

Comment:
I'm wondering whether it makes more sense to return the graph rather than the dot file. That way the server doesn't need to import routing graph printing.